licensescout-core/src/main/java/org/aposin/licensescout/configuration/FilesystemConfigFileHandler.java

```diff
@@ -196,4 +196,36 @@ public InputStream getFilteredVendorNamesInputStream() throws IOException {
         }
         return null;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean hasTemplateFile(String filename) throws IOException {
+        if (filename != null) {
+            final File templateFile = new File(filename);
+            return (templateFile.isFile() && templateFile.canRead());
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public File getTemplateFile(String filename) throws IOException {
+        if (filename != null) {
+            final File templateFile = new File(filename);
+            if (templateFile.exists()) {
+                return templateFile;
+            } else {
+                return null;
+            }
+        } else {
+            return null;
+        }
+
```

Comment:
Simplify this else chain:

```suggestion
            }
        }
        return null;
```

Reply:
Changed as requested.